test/evals/eval_data.yaml

```diff
@@ -99,6 +139,28 @@
       eval_types: [response_eval:accuracy]
       expected_response: "hosts hasn't been discovered yet."
 
+- conversation_group: oci_cluster_creation_conv
+  description: Create cluster on Oracle Cloud Infrastructure with all the information provided
+  cleanup_script: ../scripts/delete_cluster.sh
+  conversation:
+    - eval_id: create_eval_test_OCI
+      eval_query: create a new multi node cluster to OCI, with the name eval-test-oci-uniq-cluster-name, running on version 4.19.7 with the x86_64 CPU architecture, configured under the base domain example.com.
+      eval_types: [response_eval:sub-string, response_eval:accuracy, action_eval]
+      eval_verify_script: ../scripts/verify_create_eval_test_oci.sh
+      expected_keywords: ["eval-test-oci-uniq-cluster-name", "ID", "Discovery ISO", "download", "cluster"]
```

Comment:
I don't find the substring test helpful when we're also using a script and accuracy. We can probably just leave this out.